app/messages/de.json

```diff
@@ -83,7 +83,7 @@
   "components_common_interactionButton_feedback": "Feedback",
   "components_common_interactionButton_logIn": "Log in",
   "components_common_interactionButton_clear": "Entfernen",
-  "components_common_interactionButton_addUpdate": "Beitrag hinzufügen",
+  "components_common_interactionButton_addUpdate": "News hinzufügen",
```

Comment:
Beitrag is ok in this case, cause it can also be a "post" which is a news item not related to a project